sip-tbd/sip-tbd-taproot_payments.md

```diff
@@ -0,0 +1,75 @@
+
+# License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at [Creative Commons — CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/). This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+The Taproot upgrade was first proposed to the Bitcoin network in 2018 to improve efficiency. As a result, one of the core updates was a technique called “key aggregation,” which enables multi-signature transactions to commit the same amount of data to the Bitcoin blockchain as a single standard transaction. With this technique, transactions are batched into a single transaction, resulting in a lighter load. 
+
+With this technology, there are benefits to the Stacks ecosystem with respect to mining. Enabling Taproot scripts allows independent miners to pool together their mining bids as a single transaction. With that change, more independent miners can come on board to pool together for block bids. Adding the optionality to send Taproot payments to Stacks smart contracts streamlines mining pools operations within their protocol. 
+
+# Modification
+
+This SIP proposes to integrate Taproot script payments to the Stacks blockchain. Additionally, this SIP also proposes to allow the option to send Taproot script payments to Stacks Smart Contracts.
+
+# Rationale
+
+In the current state, approximately ~5 miners are operating to secure the Stacks network [1].  
+<img width="1175" alt="Onstacks Mining Stats 5-24-2022" src="https://user-images.githubusercontent.com/46361137/170338141-28ddbd5f-efe8-4e73-87b2-afb509fda8ab.png">
+                          **Source:** [OnStacks](https://app.onstacks.com/), accurate as of 24 May 2022
+
+In the event that these ~5 miners go offline, Stacks blockchain is open to vulnerabilities. Allowing independent miners to pool together through Taproot scripts further decentralizes the mining of Stacks. In general, greater miner numbers increase resiliency and decentralization of the Stacks blockchain which is a favorable outcome for the ecosystem.
+
+A new Stacks block may be mined once per Bitcoin block, amassing 144 blockers per day. To be considered for mining a block, a miner must have a block commit included in a Bitcoin block. This is determined by the amount of Bitcoin the miners have sent to be burned. Oftentimes, independent miners will not “win” the block as they do not have enough Bitcoin committed. However, with Taproot payments, independent miners can form something akin to a mining pool with a single transaction commitment. This update avoids many Bitcoin bandwidth bottleneck issues because to the Bitcoin network, the pool just looks like a single miner.
```

Comment:
To be clear, the BTC isn't burnt most of time.  It's sent to Stacks holders via PoX, and only burnt in the PoX prepare phase, and in case of emergencies where PoX cannot happen (per SIP-007).

Reply:
If I change "burned" to "committed" would that be a better word choice?